Sources/KSCrashRecording/include/KSCrash.h

```diff
@@ -127,6 +127,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)installWithConfiguration:(KSCrashConfiguration *)configuration error:(NSError **)error;
 
+/** TODO: Add docs
+ */
+- (BOOL)installReportsOnlyWithInstallPath:(nullable NSString *)installPath error:(NSError **)error;
```

Comment:
Good point. I like `setupReportStoreWithPath:error`, will update the API.

Also, it's worth checking that install later happens with the same path and log a warning in case it's different. I can imagine the use case of migration between paths, so it shouldn't be an error.